creating pull request

francis3400 opened the pull request.

francis3400 committed:
added stylelintrc.json file

francis3400 committed:
Merge pull request #2 from francis3400/feature-1

Feature 1

1GORDON reviewed (changes requested):
##
## Hi @francis3400 

Good job so far!

## Highlights!
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] Kindly follow the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) as specified in the general requirements. Make sure your branch `feature1` is merging into the `main` instead of vice-versa.
- [ ] Kindly do not merge a branch before it is approved. This is not the recommended practice.
- [ ] Kindly add a [readme](https://github.com/microverseinc/readme-template) file to your project and customize it accordingly.
- [ ] Kindly add a description to your Pull Request. At the moment your Pull Request has no description.
![describe](https://user-images.githubusercontent.com/63917725/192975685-92a6b617-0fdc-4ea3-8707-9de5ebaf95c9.PNG).

- [ ] Kindly surround these technologies with `<li>` tags and put them inside of a `<ul>` tag.
![liz](https://user-images.githubusercontent.com/63917725/192976931-6e051ace-344d-445f-b047-b62c446ca670.PNG)

- [ ] Different elements of your design are overlapping with other elements. Kindly fix this issue so that your design is a complete replica of the design provided in the Figma template.
![2](https://user-images.githubusercontent.com/63917725/192977724-96e74d60-a935-4b14-84aa-b2e090d1c9ff.PNG)

- [ ] Kindly make sure the main container of your cards has a `display: grid`.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

francis3400 committed:
Merge pull request #4 from francis3400/feature-1

updated all files

francis3400 committed:
added index and styles files

edahigure committed:
finishing title and text content on contact form

francis3400 committed:
modified index and style files

edahigure committed:
fixing button contact form

francis3400 committed:
final changes index and styles files

francis3400 committed:
Merge branch 'mobile-complete-version' of github.com:francis3400/portfolio- into mobile-complete-version

francis3400 committed:
Update README.md

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

francis3400 committed:
Merge pull request #5 from francis3400/mobile-complete-version

Mobile complete version done.

francis3400 committed:
Finished Desktop Version

francis3400 committed:
updated README

Added some changes in the readme.

francis3400 committed:
Merge branch 'main' into Portfolio-Desktop-Version

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

francis3400 committed:
added index and styles files

francis3400 committed:
finished changes

francis3400 committed:
Merge branch w'Portfolio-Desktop-Version' of github.com:francis3400/portfolio- into Portfolio-Desktop-Version

francis3400 committed:
Final Corrections

francis3400 committed:
Final Changes

francis3400 committed:
Update styles.css

francis3400 committed:
changes index

francis3400 committed:
commpleted styles

francis3400 committed:
final changes

edahigure committed:
add animation and transition

francis3400 committed:
Merge pull request #6 from francis3400/Portfolio-Desktop-Version

Portfolio desktop version

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

francis3400 committed:
Update styles.css

edahigure committed:
Update styles.css

francis3400 committed:
Merge pull request #7 from francis3400/animation-and-transitions

add animation and transition

francis3400 committed:
Final Changes Desktop Portfolio

francis3400 committed:
Merge branch 'main' into Portfolio-Desktop-Version

francis3400 committed:
Merge pull request #8 from francis3400/Portfolio-Desktop-Version

Final Changes Desktop Portfolio

francis3400 committed:
Updated README file


I updated  the readme file with the online link of my portfolio.
-I removed the linters error from the CSS and HTML file.
-I added .gitignore file to filter the unwanted files

francis3400 committed:
Merge pull request #9 from francis3400/deployment

Updated README file

francis3400 committed:
improve tags for accessibility

Itswali committed:
made html accessible

Page title -  made it accessible
The basic structure of the page - Noisssues found
Color-contrast- Colors are exactly same as figma design
Interaction- add labels / aria-label to contact form
Multimedia - No Issues found
Moving content - No issues found
Links - made it accessible using roles
Resize - No issues found
Text-heading- made it accessible
Image text alternatives- No issues found.
Page Title- Fix it according to rules for accessibility

Itswali committed:
Merge pull request #10 from francis3400/accessibility

Accessibility (add accessibility in html)

francis3400 committed:
add index and style files

francis3400 committed:
modified index files

francis3400 committed:
Merge pull request #12 from francis3400/accessibility

Accessibility

francis3400 committed:
add index files

francis3400 committed:
Update index.html to add mobile menu

francis3400 committed:
Add script.js to add mobile menu interactions

francis3400 committed:
Updated Styles.css files

amentetteh committed:
Update index.html

amentetteh committed:
Update script.js

amentetteh committed:
Update styles.css to add margin top to the menu

francis3400 committed:
Update README.md

francis3400 committed:
Merge pull request #13 from francis3400/Javascript-work

Mobile Menu Interaction Design

francis3400 committed:
Update js files for interactions

amentetteh committed:
Update index.html to load dynamically projects

amentetteh committed:
Update script.js to load dynamically projects

amentetteh committed:
Update s

francis3400 committed:
Update script.js

amentetteh committed:
Update script.js to add form validation

francis3400 committed:
Update styles.css to validate form

francis3400 committed:
Update linters.yml for html/css/js

francis3400 committed:
Update script.js

francis3400 committed:
Fixed Eslint errors

francis3400 committed:
Fixed Eslint errors

francis3400 committed:
Merge pull request #15 from francis3400/form-validation

Form validation

francis3400 committed:
Update README.md
